deno/lib/__tests__/generics.test.ts

```diff
@@ -15,6 +15,7 @@ test("generics", () => {
         nested: schema, // as z.ZodTypeAny,
       })
       .transform((data) => {
+        // biome-ignore lint/style/noNonNullAssertion: <explanation>
```

Comment:
Is this potentially covering up a bug? Should replace the placeholder with an explanation here.